### ISSUE TITLE
salt cli --summary flag broken 2015

### ISSUE THREAD

**Issue by slimg00dy** [saltstack/salt#25108]
On the latest version of salt master and salt minion, adding the --summary flag when using the CLI returns an attribute error.

```
[DEV root@salt.dev cortana]# salt "build13*" test.version --summary
build13.dev.iad.resonatedigital.net:
    2015.5.0
[ERROR   ] An un-handled exception was caught by salt's global exception handler:
AttributeError: 'str' object has no attribute 'get'
Traceback (most recent call last):
  File "/usr/bin/salt", line 10, in <module>
    salt_main()
  File "/usr/lib/python2.7/dist-packages/salt/scripts.py", line 349, in salt_main
    client.run()
  File "/usr/lib/python2.7/dist-packages/salt/cli/salt.py", line 210, in run
    self._print_returns_summary(ret)
  File "/usr/lib/python2.7/dist-packages/salt/cli/salt.py", line 242, in _print_returns_summary
    minion_ret = ret[each_minion].get('ret')
AttributeError: 'str' object has no attribute 'get'
Traceback (most recent call last):
  File "/usr/bin/salt", line 10, in <module>
    salt_main()
  File "/usr/lib/python2.7/dist-packages/salt/scripts.py", line 349, in salt_main
    client.run()
  File "/usr/lib/python2.7/dist-packages/salt/cli/salt.py", line 210, in run
    self._print_returns_summary(ret)
  File "/usr/lib/python2.7/dist-packages/salt/cli/salt.py", line 242, in _print_returns_summary
    minion_ret = ret[each_minion].get('ret')
AttributeError: 'str' object has no attribute 'get'
[DEV root@salt.dev cortana]# salt-master -version
Usage: salt-master

salt-master: error: no such option: -v
[DEV root@salt.dev cortana]# salt-master --version
salt-master 2015.5.0 (Lithium)
[DEV root@salt.dev cortana]#
```

Without it the summary flag, it works just fine.

```
[DEV root@salt.dev cortana]# salt "build13*" test.version
build13.dev.iad.resonatedigital.net:
    2015.5.0
[DEV root@salt.dev cortana]#
```

When downgraded to 2014, summary flag works again.

```
[DEV root@salt.dev cortana]# salt-master --version
salt-master 2014.7.5 (Helium)
[DEV root@salt.dev cortana]# salt "build13*" test.version --summary


-------------------------------------------
Summary
-------------------------------------------
# of Minions Targeted: 0
# of Minions Returned: 0
# of Minions Did Not Return: 0
-------------------------------------------
[DEV root@salt.dev cortana]# salt "build13*" test.version
build13.dev.iad.resonatedigital.net:
    2014.7.5
[DEV root@salt.dev cortana]#
```

**Comment by tbaker57** [comment]
I think this is a duplicate of #24111, #23890. The fix should appear in 2015.5.3 according to #24111.

cheers,
Tim

**Comment by jfindlay** [comment]
@slimg00dy, thanks for the report. The fix is in #24732, which will come out in 2015.5.3 in about a week.
